### PR TITLE
docs: corrected grammar in about.html

### PR DESCRIPTION
**Pull request by kelvinelove**
docs: I made grammar corrections as what was there wasn't sounding right

### Checks

- [ ] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [ ] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [ ] Make sure to include your GitHub username inside parentheses at the end of the PR title

Closes #

### CODE REVIEW

File: frontend/src/html/pages/about.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h2>
       You can use
       <key>tab</key>
       and
-      <key>enter</key>
+      <key>enter</key> keys
       (or just
```
Comment (Miodec) [inline frontend/src/html/pages/about.html:84]: tab and enter is meant to be a key combination, so adding "keys" here changes the meaning of the sentece.

**Comment by Miodec** [comment]
Im not sure about the started / completed / time strings. It doesnt sound incorrect to me

**Comment by kelvinelove** [comment]
> Im not sure about the started / completed / time strings. It doesnt sound incorrect to me

In english grammar, it supposed to be "total typing time" and "total tests completed" is more concise, clear and grammatically sound, instead of what's currently there, which does not follow the standard _adjective-noun order_. Although one can pick up what the titles are relating to, it's grammatically incorrect and vague, hence why I suggested something that can be easily understood, especially with people whose first language is not english @Miodec

**Comment by Miodec** [comment]
I dont see how "total tests started" would be hard to understand for a non native speaker, but fine.

I would still change the "keys" thing though. Remove it and maybe replace "and" with "+" to indicate its a key combination.

**Comment by kelvinelove** [comment]
> I dont see how "total tests started" would be hard to understand for a non native speaker, but fine.
> 
> I would still change the "keys" thing though. Remove it and maybe replace "and" with "+" to indicate its a key combination.

Okay. I've removed the `keys` part. Thank you so much @Miodec

**Pull request merged**